pkg/cloud/services/network/subnets.go

```diff
@@ -451,7 +451,6 @@ func (s *Service) getSubnetTagParams(unmanagedVPC bool, id string, public bool,
 	} else {
 		return infrav1.BuildParams{
 			ResourceID: id,
-			Additional: additionalTags,
```

Comment:
crit. There is a logic above that adds tags for CCM ,we still want those tags to be added to the unmanaged subnets so only additionalTags added by user should be removed from unmanaged subnets.

https://github.com/kubernetes-sigs/cluster-api-provider-aws/blob/7f094b6e315e54b89e2aa3385d7838244c3e2c87/pkg/cloud/services/network/subnets.go#L422

Reply:
Done